Assets/Scripts/Calendar UI.cs

```diff
@@ -6,14 +6,16 @@
 using Unity.VisualScripting;
 using UnityEngine;
 using UnityEngine.UI;
-
+using Ksen;
```

Comment:
Either all files in an assembly should have the same namespace, or they should not exist in a namespace.

Reply:
Resolved